src/App/App.js

```diff
@@ -86,35 +90,18 @@ class App extends Component {
               )}
               h
             />
-            <Route
-              path="/login"
-              render={(props) =>
-                this.state.loggedIn ? (
-                  <Redirect to="/" />
-                ) : (
-                    <Login {...props} toggleLogin={this.toggleLogin} />
-                  )
-              }
-            />
-            <Route
-              exact
-              path="/"
-              render={(props) =>
-                this.state.error > 400 ? (
-                  <Redirect to="/error" />
-                ) : (
-                    <Dashboard
-                      {...props}
-                      allMovies={this.state.movies}
-                      userRatings={this.state.userRatings}
-                      update={this.componentDidUpdate}
-                    />
-                  )
-              }
-            />
+            <Route exact path="/login">{logged}</Route>
+            <Route exact path="/">
+              <CardsContainer
+                allMovies={this.state.movies}
+                userRatings={this.state.userRatings}
+                update={this.componentDidUpdate} />
+            </Route>
+            <Route exact path="/error" component={ErrorPage} />
+            <Redirect to="/error" />
```

Comment:
This definitely cleans our app up quite a bit, glad we could slim down the passing of router props